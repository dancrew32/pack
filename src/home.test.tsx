import * as React from "react";
import * as TestRenderer from "react-test-renderer";

import Home from "./home";

test("<Home />", () => {
  const tree = TestRenderer.create(<Home />).toJSON();
  expect(tree).toMatchSnapshot();
});
