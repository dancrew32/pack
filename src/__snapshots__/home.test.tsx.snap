// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> 1`] = `
<div
  className="container"
>
  <h1>
    Hello, world!
  </h1>
  <nav>
    <a
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <a
      href="/about"
      onClick={[Function]}
    >
      About
    </a>
  </nav>
  <img
    className="container-img"
    src="test-file-stub"
  />
  <p>
    This is the home page.
  </p>
  <p>
    Magnificent.
  </p>
</div>
`;
